docker/base-envoy/Dockerfile.stripped

```diff
@@ -16,7 +16,7 @@
 # If you have to change this file, you _must_ update BASE_ENVOY_RELVER
 # in the Makefile, then run "make update-base" to build and push the
 # new image.
-FROM frolvlad/alpine-glibc:alpine-3.17_glibc-2.32
+FROM frolvlad/alpine-glibc:alpine-3.17_glibc-2.34
```

Comment:
Looks like `BASE_ENVOY_RELVER` in `_cxx/Makefile` wasn't updated when this was last updated. Instead of changing the base image here, bump `BASE_ENVOY_RELVER` and try running `make update-base` again which should push a new version of the image.

Reply:
my bad, you're right it should be updated to frolvlad/alpine-glibc:alpine-3.17_glibc-2.34, just also bump BASE_ENVOY_RELVER as well

Reply:
@haq204 - yes this is needed to build envoy.